pandas/core/frame.py

```diff
@@ -4682,17 +4684,17 @@ def f(vals):
 
     # ----------------------------------------------------------------------
     # Sorting
-
     @Substitution(**_shared_doc_kwargs)
     @Appender(NDFrame.sort_values.__doc__)
-    def sort_values(
+    def sort_values(  # type: ignore[override] # NOQA
```

Comment:
@simonjayhawkins is this the best way to do this ignore?

Reply:
my preference would be to not include the codes until the flake issues are resolved, see #29197

Reply:
@TomAugspurger Do you think we should just `# type: ignore` it unconditionally, and then change it once `pyflakes` has a new release?

Reply:
No idea.

Reply:
I would go with @simonjayhawkins here if no strong objections against; we have #30451 open anyway